drivers/net/ethernet/renesas/rswitch.c

```diff
@@ -797,6 +797,10 @@ enum rswitch_etha_mode {
 #define LTHSLP0v6 (6)
 /* L3 Routing Valid Learn */
 #define LTHRVL (BIT(15))
+#if IS_ENABLED(CONFIG_RENESAS_FWD_MON)
```

Comment:
You can define register bits regardless of CONFIG_ states.

Reply:
The redundant config was removed, so this line is not needed anymore.